src/components/net/linux-net.c

```diff
@@ -51,6 +51,11 @@ papi_vector_t _net_vector;
 #define NET_INVALID_RESULT     -1
 
 
+// The following macro follows if a string function has an error. It should 
+// never happen; but it is necessary to prevent compiler warnings. We print 
+// something just in case there is programmer error in invoking the function.
+#define HANDLE_STRING_ERROR {fprintf(stderr,"%s:%i unexpected string function error.\n",__FILE__,__LINE__); exit(-1);}
```

Comment:
I would not add `HANDLE_STRING_ERROR` anywhere. In fact, we should eventually remove it from everywhere it is used. Calling exit is not a good way of handling errors. Can you return `PAPI_EBUF` instead?